src/modals/collectible-modal/utils/get-nft-dynamic-url.ts

```diff
@@ -0,0 +1,26 @@
+import dynamicLinks from '@react-native-firebase/dynamic-links';
+
+import { TokenInterface } from 'src/token/interfaces/token.interface';
+import { DYNAMIC_LINKS_DOMAIN_URI_PREFIX } from 'src/utils/env.utils';
+import { formatImgUri } from 'src/utils/image.utils';
+
+export const getNftDynamicUrl = async (collectible: TokenInterface) =>
+  await dynamicLinks().buildShortLink(
+    {
+      link: `${DYNAMIC_LINKS_DOMAIN_URI_PREFIX}/nft?jsonData=${encodeURIComponent(JSON.stringify(collectible))}`,
+      domainUriPrefix: DYNAMIC_LINKS_DOMAIN_URI_PREFIX,
+      ios: {
+        appStoreId: '1610108763',
```

Comment:
is it public data? maybe we need to add this in env

Reply:
its public. You can take it from app store link https://apps.apple.com/us/app/temple-wallet-mobile/id1610108763